test/core/iomgr/tcp_client_posix_test.cc

```diff
@@ -214,24 +217,25 @@ void test_connect_cancellation_succeeds(void) {
 
   /* create a phony server */
   svr_fd = socket(AF_INET, SOCK_STREAM, 0);
-  GPR_ASSERT(svr_fd >= 0);
-  GPR_ASSERT(
-      0 == bind(svr_fd, (struct sockaddr*)addr, (socklen_t)resolved_addr.len));
-  GPR_ASSERT(0 == listen(svr_fd, 1));
+  ASSERT_GE(svr_fd, 0);
+  ASSERT_EQ(0,
```

Comment:
nit: most of these tests use the ordering `ASSERT_*(variable, expected_value)`. It would be nice to make that consistent throughout. See https://en.wikipedia.org/wiki/Yoda_conditions

Reply:
Done.

Reply:
Done.